motr/idx_dix.c

```diff
@@ -547,8 +547,20 @@ static void cas_next_ast(struct m0_sm_group *grp, struct m0_sm_ast *ast)
 static void dix_build(const struct m0_op_idx *oi,
 		      struct m0_dix          *out)
 {
+	unsigned int   opcode = oi->oi_oc.oc_op.op_code;
```

Comment:
We have defined:
```
#define OP_OBJ2CODE(op_obj) op_obj->oo_oc.oc_op.op_code

#define OP_IDX2CODE(op_idx) op_idx->oi_oc.oc_op.op_code
```
which can be used here.

If we don't use them, no need to define them.

Reply:
OP_IDX2CODE(op_idx) -- Already it is used in motr/idx.c file. I will assign opcode value with  OP_IDX2CODE() macro.